### PR TITLE
Implementing option "from" in fetchGames method

### PR DESCRIPTION
**Pull request by mabalashov**

### CODE REVIEW

**Comment by mabalashov** [comment]
I have to pass the latest block from fetchConditions to update statistics **only** after fetching all games data

The problems: sometimes we can face errors during fetching games data. But we are not stopping processing. In this case some games data could be missed
Ideally we need to implement full pagination

File: src/contracts/statistics.ts [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+  latestConditionsBlock: number,
+}
+
+const statistics: Statistics = {
+  latestConditionsBlock: null,
+}
+
+export const getLatestConditionsBlock = () => statistics.latestConditionsBlock
+
+export default statistics
```
Comment (pavelivanov) [inline src/contracts/statistics.ts:11]: Basically it named `stats`.

File: src/api/fetchConditions.ts [hunk 2]
```diff
@@ -19,6 +19,11 @@ export type Condition = {
   gameData: ConditionGameData
 }
 
+export type Conditions = {
+  conditions: Condition[],
+  latestBlock: number,
```
Comment (pavelivanov) [inline src/api/fetchConditions.ts:24]: remove `,` from types

File: src/api/fetchConditions.ts [hunk 3]
```diff
@@ -33,16 +38,17 @@ export type FetchConditionsProps = {
...
 
-const fetchConditions = async (props?: FetchConditionsProps): Promise<Condition[]> => {
+const fetchConditions = async (props?: FetchConditionsProps): Promise<Conditions> => {
   const { resolved = true, canceled = true } = props?.filters || {}
 
   const coreContract = getContract('core')
 
   const createdFilter = coreContract.filters.ConditionCreated()
-  const events = await coreContract.queryFilter(createdFilter)
+  const events = await coreContract.queryFilter(createdFilter, props.from)
```
Comment (pavelivanov) [inline src/api/fetchConditions.ts:51]: `props` is optional

**Review by pavelivanov — CHANGES_REQUESTED** [review]

**Pull request merged**